### PR TITLE
New refund panel

### PR DESCRIPTION
**Pull request by xseignard**
- revamped component
- connected to aragon api

### CODE REVIEW

**Comment by xseignard** [comment]
Hi @deamme could you check this and maybe provide a better design of the refund panel ?

Thanks in advance!

File: apps/aragon-fundraising/app/src/components/NewRefund/Refund.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          )
+        })}
+      {(!account || !contributions.get(account)) && <Wrapper>You don't have any contributions</Wrapper>}
+      <Info />
+    </div>
+  )
+}
+
+const Wrapper = styled.div`
+  padding-top: 10px;
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/components/NewRefund/Refund.js]: Let's get `1rem` on this one

File: apps/aragon-fundraising/app/src/components/NewRefund/Refund.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                Contribution of {formatBigNumber(c.value, decimals)} {symbol} the {new Date(c.timestamp).toLocaleDateString()}
+              </Text>
+              <Button mode="strong" wide onClick={() => handleRefund(c.vestedPurchaseId)}>
+                Refund
+              </Button>
+            </Wrapper>
+          )
+        })}
+      {(!account || !contributions.get(account)) && <Wrapper>You don't have any contributions</Wrapper>}
+      <Info />
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/components/NewRefund/Refund.js]: I would do `<Info css="margin-top: 1rem;" />`

File: apps/aragon-fundraising/app/src/components/NewRefund/Refund.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    }
+  }
+
+  return (
+    <div>
+      {account &&
+        contributions.get(account) &&
+        contributions.get(account).map(c => {
+          return (
+            <Wrapper key={c.vestedPurchaseId}>
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/components/NewRefund/Refund.js]: I would probably do:
```js
            <div key={c.vestedPurchaseId}>
              <Text css="margin: 1rem 0;">
                Contribution of {formatBigNumber(c.value, decimals)} {symbol} the {new Date(c.timestamp).toLocaleDateString()}
              </Text>
              <Button mode="strong" wide onClick={() => handleRefund(c.vestedPurchaseId)}>
                Refund
              </Button>
            </div>
```

**Comment by deamme** [comment]
@xseignard Did some minor design comments but we might not need the side panel if the user don't need to type anything.

**Pull request merged**